app/controllers/ops_controller.rb

```diff
@@ -231,7 +232,7 @@ def tree_select
     tree_selected_model
     set_active_tab(params[:id])
     session[:changed] = false
-    self.x_node = params[:id] #params[:action] == "x_show"
+    self.x_node = params[:id] # params[:action] == "x_show"
```

Comment:
please remove the extra space after the # symbol
`self.x_node = params[:id] #params[:action] == "x_show"`